src/controllers/expenseController.js

```diff
@@ -0,0 +1,94 @@
+const expensesService = require('../services/expensesService');
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const filteredExpenses = expensesService.getAllExpenses(
+    userId,
+    categories,
+    from,
+    to,
+  );
+
+  res.status(200).send(filteredExpenses);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+  const expense = expensesService.getExpenseById(id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(expense);
+};
+
+const create = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+  const user = userService.getUserById(userId);
+
+  if (!user || !spentAt || !title || !amount || !category || !note) {
```

Comment:
You are checking if the user exists and if all the required fields are present in the same condition. It would be better to separate these checks to provide more specific error messages. For example, if the user doesn't exist, you could return a 'User not found' message. If some fields are missing, you could return a 'Missing required fields' message.